content/1.7/back-office/international/locations/add-new-country.md

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
This file should be in a "countries folder" with an _index.md file and should be renamed for add-edit-country.md